lib/src/auth/exception/firebase_auth_exception_code.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:firedart/firedart.dart';
+
+/// Represents a [FirebaseAuthException] code.
+enum FirebaseAuthExceptionCode {
+  /// Indicates whether the Firebase API key is not valid.
```

Comment:
```suggestion
  /// A code that indicates the invalid Firebase API key exception.
```
or maybe this would be better: 
```suggestion
  /// An exception code indicating that the Firebase API key is not valid.
```